lib/render/vp2RenderDelegate/render_delegate.cpp

```diff
@@ -649,7 +683,37 @@ MString HdVP2RenderDelegate::GetLocalNodeName(const MString& name) const {
 MHWRender::MShaderInstance* HdVP2RenderDelegate::GetFallbackShader(
     const MColor& color) const
 {
-    return sShaderCache.GetFallbackShader(color);
+    return sShaderCache.GetFallbackShader(color, 0);
```

Comment:
This makes me feel the index could actually be an enum class. Something like this
```
enum class FallbackShaderType { Geom=0, BasisCurvesLinear, BasisCurvesCubic,Count };
```
And then count you can use to address my previous comment.

Reply:
Good point!

Reply:
Fixed in 4dd8081a75a2906ace77e578c26ceca6755fe409